pkg/operator/ceph/nfs/nfs.go

```diff
@@ -48,8 +48,8 @@ type daemonConfig struct {
 }
 
 // Create the ganesha server
-func (r *ReconcileCephNFS) upCephNFS(n *cephv1.CephNFS, oldActive int) error {
-	for i := oldActive; i < n.Spec.Server.Active; i++ {
+func (r *ReconcileCephNFS) upCephNFS(n *cephv1.CephNFS) error {
+	for i := 0; i < n.Spec.Server.Active; i++ {
```

Comment:
Good catch, we don't need to compare with the old CR value, we can just proceed with everything again to ensure the exact same state.